### PR TITLE
Add chain label to Nitro metrics

### PR DESCRIPTION
**Pull request by lambchr**
[ticket](https://www.notion.so/arbitrum/Blockscout-alerts-9dfadf0f3f13438993adb425bbfca721?pvs=4)

Adding a chain label to the Nitro, Relay and LB service monitors. This will enable us to compare metrics within a chain against each other.

### CODE REVIEW

File: charts/nitro/templates/servicemonitor.yaml [hunk 1]
```diff
@@ -27,6 +27,10 @@ spec:
       tlsConfig: 
         {{- toYaml .Values.serviceMonitor.tlsConfig | nindent 8 }}
       {{- end }}
+      relabelings:
+        - targetLabel: chain
+          replacement: {{ .Values.chain.name }}
```
Comment (a-thomas-22) [inline charts/nitro/templates/servicemonitor.yaml:32]: Nitro can have the chain name set with `configmap.data.chain.name` Lets default to that if its set and allow it to be overridden.

File: charts/das/values.yaml [hunk 2]
```diff
@@ -1,5 +1,9 @@
 ## @section DAS Deployment Options
 
+## @param chain.name Name of the chain
+chain:
```
Comment (a-thomas-22) [inline charts/das/values.yaml:4]: lets nest this value under the servicemonitor top level key. Have the relabeling be conditional only if the chain name is set or have a some kind of `enabled` key for this feature. If the relabeling is conditional if the chain name is set, have the default be an empty string.

File: charts/nitro/values.yaml [hunk 3]
```diff
@@ -1,5 +1,9 @@
 ## @section Nitro Deployment Options
 
+## @param chain.name Name of the chain
+chain:
```
Comment (a-thomas-22) [inline charts/nitro/values.yaml:4]: lets nest this value under the servicemonitor top level key. Have the relabeling be conditional only if the chain name is set here or with the `configmap.data.chain.name` or have a some kind of enabled key for this feature. If the relabeling is conditional if the chain name is set, have the default be an empty string.

File: charts/relay/values.yaml [hunk 4]
```diff
@@ -1,5 +1,9 @@
 ## @section Relay
 
+## @param chain.name Name of the chain
+chain:
```
Comment (a-thomas-22) [inline charts/relay/values.yaml:4]: lets nest this value under the servicemonitor top level key. Have the relabeling be conditional only if the chain name is set or have a some kind of enabled key for this feature. If the relabeling is conditional if the chain name is set, have the default be an empty string.

**Review by a-thomas-22 — CHANGES_REQUESTED** [review]
The changes center around making it clear what the value does. Since it is specifically related to the servicemonitor, I think it makes more sense to nest it there. And to take advantage of the nitro chain name if set already(but allow it to be overriden).

**Review by bhuddlestonocl — COMMENTED** [review]
Should these changes additionally be added to `charts-publish/nitro`?

**Comment by a-thomas-22** [comment]
@bhuddlestonocl 
> Should these changes additionally be added to `charts-publish/nitro`?

`charts-publish/nitro` should be deprecated in favor of `charts-publish/offchain-nitro` except for stylus testnet(which is on its own branch). This was done in https://github.com/OffchainLabs/sre-helm-charts/pull/1286

That new offchain-nitro uses this as a dependency so we should only need to update it here and then add whatever global overrides we want in `offchain-nitro`

**Pull request merged**